asr/logging.py

```diff
@@ -0,0 +1,190 @@
+import json
+from datetime import datetime
+from pathlib import Path
+
+
+import numpy as np
+
+
+def read_log(log_fp):
+    """Read log file.
+
+    Arguments
+    ---------
+    log_fp: str, pathlib.Path
+        File path of the log file.
+
+    Returns
+    -------
+    Logger:
+        A Logger object with logs.
+    """
+    log_fp_path = Path(log_fp)
+
+    if log_fp_path.is_file() and log_fp_path.suffix == ".json":
+
+        with open(log_fp_path, "r") as f:
+
+            log = Logger()
+            log._log_dict = json.load(f)
+
+            return log
+
+    else:
+        raise ValueError(f"Incorrect file path {log_fp}")
+
+
+def read_logs_from_dir(log_dir, prefix=None):
```

Comment:
How about prefix="", then by default it will use all files in the directory?

Reply:
This is a bug. `""` will fail as well atm. Will fix this one.